.github/workflows/tests.yaml

```diff
@@ -0,0 +1,15 @@
+name: Docker Compose Actions Workflow
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
Currently this takes about 5~ minutes to run. We might think about adding caching to it.

Reply:
deferring to later